boards/cc2650stk/include/periph_conf.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2016 Leon George
+ * (who won't go after you should you copy any part of this file ;-)
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+/**
+ * @ingroup     boards_cc2650stk
+ * @{
+ *
+ * @file
+ * @brief       peripheral MCU configuration for the CC2650STK board
+ *
+ * @author      Leon M. George <leon@georgemail.eu>
+ */
+
+#ifndef PERIPH_CONF_H_
+#define PERIPH_CONF_H_
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name timer peripheral configuration
+ * @{
+ */
+#define TIMER_NUMOF         GPT_NUMOF
+#define TIMER_0_EN          1
+#define TIMER_1_EN          1
+#define TIMER_2_EN          1
+#define TIMER_3_EN          1
+
+#define TIMER_0_IRQn_1      GPTIMER_0A_IRQN
+void isr_timer0_chan0(void);
+#define TIMER_0_ISR_1       isr_timer0_chan0
+
+#define TIMER_1_IRQn_1      GPTIMER_1A_IRQN
+void isr_timer1_chan0(void);
+#define TIMER_1_ISR_1       isr_timer1_chan0
+
+#define TIMER_2_IRQn_1      GPTIMER_2A_IRQN
+void isr_timer2_chan0(void);
+#define TIMER_2_ISR_1       isr_timer2_chan0
+
+#define TIMER_3_IRQn_1      GPTIMER_3A_IRQN
+void isr_timer3_chan0(void);
+#define TIMER_3_ISR_1       isr_timer3_chan0
+
+static const timer_conf_t timer_conf[] = {
+    {
+        GPT0,
+        TIMER_0_IRQn_1,
+        TIMER_0_ISR_1,
+    },
+    {
+        GPT1,
+        TIMER_1_IRQn_1,
+        TIMER_1_ISR_1,
+    },
+    {
+        GPT2,
+        TIMER_2_IRQn_1,
+        TIMER_2_ISR_1,
+    },
+    {
+        GPT3,
+        TIMER_3_IRQn_1,
+        TIMER_3_ISR_1,
+    },
+};
+/** @} */
+
+
+/**
+ * @name UART configuration
+ * @{
+ */
+#define UART_NUMOF          1
+
+#define UART_0_EN           1
+
+/* UART 0 device configuration */
+#define UART_0_IRQ          UART0_IRQN
+#define UART_0_ISR          isr_uart
+/* UART 0 pin configuration */
+#define UART_0_RX_DIO       28
+#define UART_0_TX_DIO       29
+/** @} */
```

Comment:
the CPU has more than 1 UART, right? So you should use a struct based configuration here as well.